examples/wasmer.sh/index.ts

```diff
@@ -0,0 +1,85 @@
+import "xterm/css/xterm.css";
+
+import { SpawnConfig, Tty, Wasmer, init } from "@wasmer/wasix";
+import { Terminal } from "xterm";
+
+const encoder = new TextEncoder();
+
+const packageName = "sharrattj/bash";
+const args: string[] = [];
+const uses: string[] = ["sharrattj/coreutils"];
+
+async function main() {
+    await init();
+
+    const term = new Terminal({ cursorBlink: true, convertEol: true });
+
+    const element = document.getElementById("app")!;
+    term.open(element);
+
+    term.writeln("Starting...");
+    const wasmer = new Wasmer();
+
+    // Attach the TTY
+    const tty = new Tty();
+    tty.state = {...tty.state, cols: term.cols, rows: term.rows};
+    term.onResize(({cols, rows}) => {
+        tty.state = {...tty.state, cols, rows};
+    });
+    wasmer.runtime().set_tty(tty);
+
+    while (true) {
+        await runInstance(term, wasmer, packageName, { args, uses });
+        term.writeln("Rebooting...");
```

Comment:
When a reboot can happen? (basically, when runInstance is done? Once the process ends? (for example in bash that would be just calling exit, or in python `os.exit()` ?